### PR TITLE
Add the documentation about the OS supported by uforge-sync

### PR DESCRIPTION
**Pull request by cinlloc**

### CODE REVIEW

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 1]
```diff
@@ -7,7 +7,12 @@ Synchronize Target Environment with Scanned Machine
 
 At the end of the migration process, you can synchronize the running migrated instance with the source machine data.
 
-.. warning:: This feature is only available for CentOS scans without overlay.
+.. warning:: This feature is only available for scans without overlay of following Operating Systems:
+
+  * CentOS 6, 7
+  * Debian 8
+
+  For other Linux Operating Systems, you can use the uforge-sync tool but result is not guaranted.
```
Comment (segalaj) [inline end-user/en/source/pages/migration/data-synchronization.rst]: Are we sure we want to expose it?

File: admin/en/source/pages/changelog.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ Release Date: 2017-10-16
 
 New Features
 ~~~~~~~~~~~~
-* Using uforge-sync binary, users can now synchronize the target environment with a CentOS scan (without overlay).
+* Using uforge-sync binary, users can now synchronize the target environment with scans without overlay of CentOS 6, 7 and Debian 8 systems.
```
Comment (segalaj) [inline admin/en/source/pages/changelog.rst]: What do you think of:
Using uforge-sync binary, users can now synchronize the target environment with a scan of CentOS 6, 7 or Debian 8 systems (without overlay).

**Review by segalaj — CHANGES_REQUESTED** [review]

File: admin/en/source/pages/changelog.rst (hunk 2 above)
Comment (cinlloc) [reply admin/en/source/pages/changelog.rst]: Or maybe `Using uforge-sync binary, users can now synchronize the target environment with scans (without overlay) of CentOS 6, 7 and Debian 8 systems.`. We need a third advice.

Comment (emuus) [reply admin/en/source/pages/changelog.rst]: I prefer Clement's option

File: end-user/en/source/pages/migration/data-synchronization.rst (hunk 1 above)
Comment (emuus) [reply end-user/en/source/pages/migration/data-synchronization.rst]: I agree... if the result is not guaranteed (note spelling) then do not state it

**Review by emuus — CHANGES_REQUESTED** [review]

**Review by segalaj — APPROVED** [review]

**Comment by cinlloc** [comment]
Transferred to https://github.com/usharesoft/appcenter-docs/pull/483

**Pull request closed without merging**